### PR TITLE
Update some Otto documentation. Fix packages.

### PR DESCRIPTION
**Pull request by hblake316**
Remove skill name from Otto's Add Daily Job Skill page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,6 @@
     ]
   },
   "volta": {
-    "node": "16.20.2"
+    "node": "20.16.0"
```
Comment (KennyJJMa) [inline package.json:42]: Should these be updated?

**Review by KennyJJMa — COMMENTED** [review]
Are we upgrading node versions here? Are there any explicit reasons to be moving away from the stable v16?

**Comment by hblake316** [comment]
> Are we upgrading node versions here? Are there any explicit reasons to be moving away from the stable v16?

Without the update, the build fails. There is a package, cheerio, that requires node >= 18.17. I chose to update to node 20 because it is stable and could buy us some time before needing to update node again.

**Review by KennyJJMa — DISMISSED** [review]

**Comment by hblake316** [comment]
Looks like there is a breaking change in a package called cheerio that is required by docusaurus-search-local. Cheerio was updated 4 days ago, maybe there is a bug?

If we do not use the docusaurus-search-local package, we can bypass this. So far, this is the only fix I've found. I'm not sure what the implications are of not using docusaurus-search-local. 

The upgrade to node 18.17.0 is no longer required without cheerio, but I left it in place as it seems to be working fine.

https://cheerio.js.org/blog/cheerio-1.0
⁠https://github.com/cmfcmf/docusaurus-search-local

Here are some potential replacements for docusaurus-search-local:
- https://docusaurus.io/community/resources#search

**Review by rweesner — APPROVED** [review]

**Pull request merged**